src/navigation/CustomDrawerContent.tsx

```diff
@@ -145,6 +145,20 @@ const CustomDrawerContent = ( { state, navigation, descriptors }: Props ) => {
         onPress: ( ) => setShowConfirm( true )
       };
     }
+
+    if ( items.logout === undefined && !currentUser ) {
+      items.login = {
+        label: t( "LOG-IN" ),
+        icon: "door-enter",
+        style: {
+          opacity: 0.5,
```

Comment:
According to the [designs](https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=3773-22107&t=xvLyzfN2P7C9d2d9-11), the icon should be green and the text black.